<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWidget</class>
 <widget class="QWidget" name="SettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="verticalGroupBox">
     <property name="title">
      <string>Theme</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>App Theme</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="themeCombo">
          <item>
           <property name="text">
            <string>Light</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Dark</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="verticalGroupBox_3">
     <property name="title">
      <string>Notifications</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Notification type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="notificationCombo">
          <item>
           <property name="text">
            <string>Native notification</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Customized notification</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="tryNotification">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Try</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Check for new snaps </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="autoRefreshInterval">
          <property name="suffix">
           <string> minutes</string>
          </property>
          <property name="prefix">
           <string>after every </string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>2000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="verticalGroupBox_4">
     <property name="title">
      <string>Other</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Run app at system startup</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="runAtStartUp">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Snap Info button click action:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="infoActionCombo">
          <item>
           <property name="text">
            <string>Open Store webpage</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Open Softwares App</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Close button action:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="closeButtonActionCombo">
          <item>
           <property name="text">
            <string>Minimize to tray</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Quit App</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="reset">
     <property name="text">
      <string>Restore default settings</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/refresh-line.png</normaloff>:/icons/refresh-line.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
